Druid/DRCogitCodeGenerator.class.st

```diff
@@ -905,13 +905,6 @@ DRCogitCodeGenerator >> visitMultiplyOverflow: aDRMultiplyOverflow inBranch: aDR
 	^ aDRBranchIfCondition
 ]
 
-{ #category : #visiting }
-DRCogitCodeGenerator >> visitMultiplyOverflowJump: aDROverflowJump [ 
-	
-	self generateJump: #JumpMultiplyOverflow: target: aDROverflowJump trueBranch.
-	self generateJump: #JumpMultiplyNoOverflow: target: aDROverflowJump falseBranch.
-]
-
```

Comment:
Why was this removed? STRANGEEE

Reply:
Because now all the logic is in [this awful method](https://github.com/Alamvic/druid/blob/9a5e22191191cd8fef108d2b18f6976576f32d96/Druid/DRCogitCodeGenerator.class.st#L44)

Reply:
you are weak